src/js/components/ContainerSettingsComponent.jsx

```diff
@@ -188,24 +224,32 @@ var ContainerSettingsComponent = React.createClass({
   },
 
   render: function () {
+    var state = this.state;
     var model = this.props.model;
     var errors = this.props.errors;
+    var network;
+
+    try {
+      network = model.container.docker.network;
+    } catch (e) {
+      network = "";
```

Comment:
What if we did 

```
var network = "";
try {
   network = model.container.docker.network;
} catch(e) {};
```
(and same thing in `getVolumeRow()` above)
so that the default case is clearer to figure out?